Mrs_Cake/_Mrs_Cake_MrsCakeDbContextDataSet.Designer.cs

```diff
@@ -0,0 +1,4963 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
@Alina-dev-front  or @abdelilahAchir Do I need to review this code? As this code came from tool?